apps/src/templates/rubrics/AiAssessmentBox.jsx

```diff
@@ -27,15 +32,21 @@ export default function AiAssessmentBox({
     }
   };
 
-  const studentAchievment = () => {
-    const assessment =
-      aiUnderstandingLevel >= RubricUnderstandingLevels.CONVINCING
-        ? i18n.aiAssessmentDoesMeet()
-        : i18n.aiAssessmentDoesNotMeet();
-    return i18n.aiStudentAssessment({
-      studentName: studentName,
-      understandingLevel: assessment,
-    });
+  const studentAchievement = () => {
+    if (hasAiInfo) {
+      const assessment =
+        aiUnderstandingLevel >= RubricUnderstandingLevels.CONVINCING
+          ? i18n.aiAssessmentDoesMeet()
+          : i18n.aiAssessmentDoesNotMeet();
+      return i18n.aiStudentAssessment({
+        studentName: studentName,
+        understandingLevel: assessment,
+      });
+    } else if (!studentSubmitted) {
+      return i18n.aiEvaluation_student_must_submit();
+    } else {
+      return i18n.aiEvaluation_not_run();
```

Comment:
I think we are going to need the same 3 cases as we are handling here: https://github.com/code-dot-org/code-dot-org/blob/6c6897ec21ee803689035f609983c633beb73b5f/apps/src/templates/rubrics/RubricSettings.jsx#L69-L78

assuming you update the value of `aiEvaluation_student_must_submit` to match its name, that would mean  adding a case for "run but already evaluated".